### PR TITLE
Fixes #333 Access docs generation form after signing in

### LINKED ISSUE #333: Access docs generation form after signing in to CI
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

User would have to Log out of the CI dashboard to access the docs generation form to preview and download docs.
### Expected behaviour

Make the form available even after the user is signed in.

### PR DESCRIPTION
**Pull request by imujjwal96**
## Description

* Change sign in button to dashboard link if logged in
* Change sign in request to post
## Related Issue

issue #333 
## How Has This Been Tested?

Tested locally
## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -6,12 +6,14 @@ var validation = require("../public/scripts/validation.js");
...
+  var isLoggedIn = false;
   if (req.user) {
-    return res.redirect('/dashboard');
+    isLoggedIn = true;
   }
 
   res.render("index", {
-    title: "Yaydoc | Automatic Documentation Generation and Deployment"
+    title: "Yaydoc | Automatic Documentation Generation and Deployment",
+    isLoggedIn: isLoggedIn,
```
Comment (sch00lb0y) [inline routes/index.js]: remove the `,`

Comment (imujjwal96) [inline routes/index.js]: There is no harm in using trailing commas in object literals. It does not raise any error
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

**Comment by imujjwal96** [comment]
Why does Github dismiss reviews with only rebasing >.< @pri22296 @sch00lb0y

**Comment by pri22296** [comment]
No idea

**Comment by imujjwal96** [comment]
@pri22296 @sch00lb0y Please review again!

**Comment by pri22296** [comment]
Can you share a deployment?

**Comment by imujjwal96** [comment]
@pri22296 https://yaydoc7.herokuapp.com

File: routes/auth.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ var express = require("express");
 var router = express.Router();
 var passport = require("passport");
 var validation = require("../public/scripts/validation.js");
+var crypter = require("../util/crypter.js");
```
Comment (pri22296) [inline routes/auth.js]: You are not using `crypter` in `auth.js`. Any reason for adding this line?

Comment (imujjwal96) [inline routes/auth.js]: i was using it before, but then i changed it. Forgot cleaning up.

**Comment by pri22296** [comment]
@imujjwal96 Rebase with upstream

**Comment by imujjwal96** [comment]
@pri22296 made changes.

**Pull request merged**